### DISCUSSION TITLE
请问下，现在还能用于竟斗云吗？

### DISCUSSION THREAD

**Discussion by bamlan** [Boos4721/openwrt#79] [Q&A/答疑解惑] [answered]
去年年底编译的选择P&W R619AC 128M，能成功编译，刷上后系统内核是5.10.x的，但是WAN口不能用，pppoe拨号失败，使用光猫拨号后WAN口使用DHCP也连接不上。后面退回5.4.x版本正常。
现在编译时，配置除了默认的，增加了ipv6helper这个包，然后luci加了几个app,未改动其它了，编译提示错误：
checking whether the compiler accepts -Wno-packed-not-aligned... yes
checking for library containing argp_parse... -largp
checking for library containing fts_close... no
configure: error: in `/mnt/docker/lede_boos/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/elfutils-0.186':
configure: error: failed to find fts_close
See `config.log' for more details
make[3]: *** [Makefile:118: /mnt/docker/lede_boos/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/elfutils-0.186/.configured_a17fb5ef857664f03cd0ce37cc5ea591] Error 1
make[3]: Leaving directory '/mnt/docker/lede_boos/package/libs/elfutils'
time: package/libs/elfutils/compile#5.35#0.59#6.75
    ERROR: package/libs/elfutils failed to build.
make[2]: *** [package/Makefile:116: package/libs/elfutils/compile] Error 1
make[2]: Leaving directory '/mnt/docker/lede_boos'
make[1]: *** [package/Makefile:110: /mnt/docker/lede_boos/staging_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/stamp/.package_compile] Error 2
make[1]: Leaving directory '/mnt/docker/lede_boos'
make: *** [/mnt/docker/lede_boos/include/toplevel.mk:230: world] Error 2

**Comment by Boos4721** [comment, marked as answer]
I`ve sold this router.
